VERSION

```diff
@@ -1 +1 @@
-1.25.0
+1.26.0
```

Comment:
This may need to be updated if #205 is merged first. Also the changelog should be updated accordingly.